README.md

```diff
@@ -39,6 +40,14 @@ https://firebase.google.com.
 * Import the firebase-android-sdk gradle project into Android Studio using the
   **Import project(Gradle, Eclipse ADT, etc.** option.
 
+## Building
+
+Building the Firebase SDKs can be performed by invoking the following on the
+command line:
+```bash
+./gradlew :<firebase-project>:assemble`
```

Comment:
I am wondering if `assembling` a stand along binary of the sdk is useful in practice, since the resulting binaries are not hermetic and require that its' dependencies be wired in.
wdyt?